src/API.js

```diff
@@ -1,48 +1,25 @@
-import axios from "axios"
+import axios from 'axios'
 
 const API = {
-    SERWER: "http://localhost:3000",
-    TOKEN: "0123456789",
+  baseHeaders: {
+    headers: {
+      Authorization: localStorage.getItem(process.env.REACT_APP_TOKEN_NAME) // process.env.REACT_APP_TOKEN
+    }
+  },
 
-    getDepartaments () {
-        return axios.get(`${API.SERWER}/api/department`, {
-            headers: {
-                Authorization: API.TOKEN
-            }
-        })
-        .then(response => response)
-        .catch(error => {
-            let errorMessage;
-            if (error.response) {
-                errorMessage = "404"
-              } else if (error.request) {
-                errorMessage = error.request
-              } else {
-                errorMessage = error.message
-              }
-              return errorMessage;
-        })
-    },
+  getDepartaments() {
+    if (localStorage.getItem(process.env.REACT_APP_TOKEN_NAME)) {
```

Comment:
you should not check for it:
for most cases it is enough just to provide the token and backend will do the rest